### PR TITLE
Update company introduction

### PR DESCRIPTION
**Pull request by landryroni**
Updated company introduction by changing the way the services were presented.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -135,13 +135,14 @@ <h4>
                 </h4>
                 <section class="presentation">
                         <p>
-                            Guangzhou Fionaa Company Limited., main activity scope is import and export business which involve sourcing, purchasing, quality control and International-Shipping.
-                            The company can do both wholesale and retail trade;
+                            Guangzhou Fionaa Company Limited., activity scope is import and export business trade which involve:
+                            Gathering Information; Factories Reasearch; Negotiation & Quoting; Sampling; Purchasing; Quality Control;
```
Comment (krylovaye2017) [inline src/index.html:139]: You can add prototyping

File: src/index.html [hunk 2]
```diff
@@ -135,13 +135,14 @@ <h4>
...
-                            The company can do both wholesale and retail trade;
+                            Guangzhou Fionaa Company Limited., activity scope is import and export business trade which involve:
+                            Gathering Information; Factories Reasearch; Negotiation & Quoting; Sampling; Purchasing; Quality Control; 
+                            Shipping and Loacl delivery.The company can do both wholesale and retail trade;
                         </p> 
 
                         <p>
                             With more than 13years of experience our company team has a certain scale and strength.
-                            We mainly export a huge variety of goods such as machinery, building materials, lighting, furniture, home appliance ...
+                            We mainly export a huge variety of goods such as machinery, building materials, lighting, furniture, home appliances ...
```
Comment (krylovaye2017) [inline src/index.html:145]: home appliances and decorations could be better

**Review by krylovaye2017 — CHANGES_REQUESTED** [review]
I did make some comments on worlds to add

File: src/index.html (hunk 2 above)
Comment (landryroni) [reply src/index.html:145]: Here also decorations was added.

**Pull request merged**